### ISSUE TITLE
Cannot get translation dynamically by key

### ISSUE THREAD

**Issue by DirtyNative** [slang-i18n/slang#261]
I cannot access my translations via:

``` dart
translations['my.key']
```

as the compiler tells me: `The operator '[]' isn't defined for the type 'TranslationsUiEn'.
Try defining the operator '[]'.`

Within the documentation it is clearly stated like so. [See this](https://github.com/slang-i18n/slang?tab=readme-ov-file#-dynamic-keys--flat-map)

Also found this thread:
https://github.com/slang-i18n/slang/issues/50

I am not sure if it is actually a bug, my stupidity, or that the package changed some things in previous versions and the documentation is not up to date anymore.

**Comment by Tienisto** [comment]
This is not the root variable. You have `TranslationsUiEn` but you need `Translations`/`TranslationsEn`

**Comment by DirtyNative** [comment]
True, this was caused by a lot of try and error. The real error is `The operator '[]' isn't defined for the type 'Translations'.
Try defining the operator '[]'`

**Comment by Tienisto** [comment]
Can you show me your config?
